### ISSUE TITLE
Unrecognized field in SOAP configuration

### ISSUE THREAD

**Issue by Parampula** [outofcoffee/imposter#277]
I am trying to customise the mock response, based on a given SOAP input.
Following your official documentation (https://medium.com/@outofcoffee/mocking-soap-web-services-with-imposter-da8e9666b5b4), 
I came across the following configuration possibility:

```
plugin: soap
wsdlFile: petstore.wsdl
resources:
  - operation: getPetById
    requestBody:
      xPath: "/env:Envelope/env:Body/pets:getPetByIdRequest/pets:id"
      value: "3"
      xmlNamespaces:
        env: "http://www.w3.org/2001/12/soap-envelope"
        pets: "urn:com:example:petstore"
    response:
      staticFile: getPetByIdResponse.xml
```

But when i try to start stub server, error Unrecognized field "requestBody" occurs.
Analyzing the code, i noted that "requestBody" is not part of available properties ("security", "response", "path", "capture", "operation", "soapAction", "binding").

Is that functionality temporarily removed, because of some issues that you are working on, or documentation is not updated ?

**Comment by outofcoffee** [comment]
Hi @Parampula. Thank you for the bug report.

This has been fixed in [3.5.1](https://github.com/outofcoffee/imposter/releases).

Example added here: https://github.com/outofcoffee/imposter/tree/main/examples/soap/request-body-match

Please let me know if this fixes the issue you were seeing.

**Comment by outofcoffee** [comment]
Closing a resolved. Thank you again for the report. Please feel free to raise a new issue if there's anything further.
